composer.json

```diff
@@ -1,27 +1,36 @@
 {
     "name": "sminnee/silverstripe-apikey",
     "description": "API Key management for SilverStripe",
-    "type": "silverstripe-vendormodule",
-    "require": {
-        "silverstripe/framework": "^4.0",
-        "silverstripe/graphql": "^1.0",
-        "silverstripe/vendor-plugin": "^1.0"
-    },
     "license": "BSD-3-Clause",
     "authors": [
         {
             "name": "Sam Minnee",
             "email": "sam@silverstripe.com"
         }
     ],
+    "type": "silverstripe-vendormodule",
+    "require": {
+        "silverstripe/framework": "^4.0",
```

Comment:
Since you're bumping the GraphQL version we need to bump the framework version as well so users won't install this on 4.0 or 4.1

Reply:
This actually works with 1 and 2 so I've just broadened the requirements

Reply:
Ok great